app/javascript/components/recipes/new/list.vue

```diff
@@ -135,6 +136,11 @@ export default {
       this.steps[index] = newText;
       this.$forceUpdate();
     },
+    delIngredient(ingredient) {
+      const index = this.elements.indexOf(ingredient);
+      this.elements.splice(index, 1);
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected mutation of "elements" prop  vue/no-mutating-props